.gitlab-ci.yml

```diff
@@ -808,40 +773,48 @@ iot_agent_deb-x64:
     paths:
       - $OMNIBUS_PACKAGE_DIR
 
+iot_agent_deb-x64:
+  stage: package_build
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/deb_x64:$DATADOG_AGENT_BUILDIMAGES
+  tags: [ "runner:main", "size:2xlarge" ]
+  needs: [ "fail_on_non_triggered_tag"]
+  variables:
+    AWS_CONTAINER_CREDENTIALS_RELATIVE_URI: /credentials
```

Comment:
I see you removed this line from all iot builds except this one, is this one needed?